pkg/asset/manifests/proxy.go

```diff
@@ -2,6 +2,7 @@ package manifests
 
 import (
 	"fmt"
+	"github.com/openshift/installer/pkg/types/openstack"
```

Comment:
this import needs to be moved to the correct location. Looks like you need to simply squash this change.